benchmarks/operators/score.py

```diff
@@ -0,0 +1,267 @@
+# Copyright (c) 2019, Microsoft CORPORATION. All rights reserved.
```

Comment:
Does this file not need BSD/NVid license?  I know you wrote most of this file, just double checking

Reply:
Yes this file contains only our stuff.